OneBlink.SDK.Tests/OrganisationsClientTest.cs

```diff
@@ -46,8 +46,22 @@ public void throws_error_if_keys_empty()
         public async void can_upload_asset()
         {
             OrganisationsClient organisationsClient = new OrganisationsClient(ACCESS_KEY, SECRET_KEY, TenantName.ONEBLINK_TEST);
-            var response = await organisationsClient.UploadAsset("data", "text/plain", "test.txt");
-            Assert.NotNull(response);
+
+            using (var stream = GenerateStreamFromString("data"))
+            {
+                var response = await organisationsClient.UploadAsset(stream, "text/plain", "test.txt");
+                Assert.NotNull(response);
+            }
+        }
+
+        public static Stream GenerateStreamFromString(string s)
```

Comment:
What do you think about doing this inside our method? so we can still pass it a string?

Reply:
It needs to be the binary data not a string